includes/DataObjects/Request.php

```diff
@@ -263,12 +271,12 @@ public function hasComments()
 			return true;
 		}
 
-		$commentsQuery = $this->dbObject->prepare("SELECT COUNT(*) as num FROM comment where request = :id;");
-		$commentsQuery->bindValue(":id", $this->id);
+		if (is_null(self::$commentsQuery)) self::$commentsQuery = $this->dbObject->prepare("SELECT COUNT(*) as num FROM comment where request = :id;");
```

Comment:
It's essentially the same commands but it's being sent to a static variable in the class.  This checks if the statement is already prepared.